### PR TITLE
DB: Database connection setup

### PR DESCRIPTION
**Pull request by MHA10**
- Database connection setup
- Postgres
- Docker
- TypeORM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,146 +1,154 @@
...
- ## Running the project locally for development
+## Running the project locally for development
 
 Install the dependencies:
 
- ```bash
- yarn
- ```
+```bash
+npm
```
Comment (MHassanTariq) [inline README.md]: npm _install_, I guess.

File: README.md [hunk 2]
```diff
@@ -1,146 +1,154 @@
...
+```
 
- ### Create a new migration
+### Create a new migration
 
- ```bash
- TableName=<name> yarn migration:create
- ```
+```bash
+TableName=<name> npm run migration:create
```
Comment (MHassanTariq) [inline README.md]: Please reconfirm this.

File: package.json [hunk 3]
```diff
@@ -13,6 +13,22 @@
...
+    "local-migration:show": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js migration:show -d ./src/config/datasource.ts",
+    "prod-migration:show": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js migration:show -d ./dist/config/datasource.js",
+    "local-migration:schema-sync": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js schema:sync -d ./src/config/datasource.ts",
+    "prod-migration:schema-sync": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js schema:sync -d ./dist/config/datasource.js",
+    "local-migration:schema-log": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js schema:log -d ./src/config/datasource.ts",
+    "prod-migration:schema-log": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js schema:log -d ./dist/config/datasource.js",
+    "local-migration:schema-drop": "NODE_ENV=dev ts-node ./node_modules/typeorm/cli.js schema:drop -d ./src/config/datasource.ts",
+    "prod-migration:schema-drop": "NODE_ENV=production ts-node ./node_modules/typeorm/cli.js schema:drop -d ./dist/config/datasource.js",
```
Comment (MHassanTariq) [inline package.json]: Can we reduce the size of scripts and keep only the important ones?

File: src/config/ormconfig.ts [hunk 4]
```diff
@@ -1,8 +1,4 @@
 import { DataSourceOptions } from 'typeorm';
-import { config } from 'dotenv';
-
-// Load environment variables from a .env file if necessary
-config();
```
Comment (MHassanTariq) [inline src/config/ormconfig.ts:5]: Please reconfirm the correct way of loading config from .env.

**Review by MHassanTariq — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MHA10) [reply README.md]: yup

File: README.md (hunk 2 above)
Comment (MHA10) [reply README.md]: works well

File: package.json (hunk 3 above)
Comment (MHA10) [reply package.json]: yup, sure

File: src/config/ormconfig.ts (hunk 4 above)
Comment (MHA10) [reply src/config/ormconfig.ts:5]: yeah nest.config.ts got this covered

**Comment by MHA10** [comment]
@MHassanTariq 
I'll improve and clean the package.json commands, then you can review those.

File: package.json (hunk 3 above)
Comment (MHA10) [reply package.json]: @MHassanTariq have a look now.

**Review by MHassanTariq — APPROVED** [review]

**Pull request merged**